usage/jsgui/src/main/webapp/assets/js/view/application-add-wizard.js

```diff
@@ -85,6 +85,13 @@ define([
         if (entity.config && _.size(entity.config)) result["brooklyn.config"] = entity.config;
         return result;
     }
+    function getConvertedConfigValue(value) {
+        try {
+            return $.parseJSON(value);
+        } catch (e) {
+            return value;
```

Comment:
I have examples of that. One of my clients uses `String` configkey only because of this issue.

Does it really matter to check at this stage the config key is != enum, string ? The CAMP Java parser will convert to the correct type afterward. 

For example, I have the following enum:
```
public enum Product {
    A,
    B,
    C;
}
```
and configuration key:
```
ConfigKey<Product> PRODUCT = ConfigKeys.newConfigKey(Product.class, "product", "My product", Product.A);
```
If I put this in the UI `product: B`, the method `getConvertedConfigValue()` will return 'B' as a JS `String`. But the CAMP parser will still try to match that to my enum by calling `Product.valueOf("B");` which will return the valid result.